test/core/camera/CameraDataJSON.dyn

```diff
@@ -0,0 +1,146 @@
+{
+  "Uuid": "d8af6cb8-3bf4-4022-a8fe-00d118ced0df",
```

Comment:
because the JSON format is changing right now, what about if you test opens an XML file, then saves it to JSON and then opens it and compares, that way you dont need to add a new file, just pick any old xml dyn.